File: IntArray_Sort.cs
namespace sol
{
    class Sol
    {

        public static int[] exampleArr =
        {
            1, 9, 4, 2, 19, 12
        };

        public static int[] sortIntArray(int[] unsortedArray)
        {
            int flags = 0;
            int temp;
            for(int i = 0; i < unsortedArray.Length; i++)
            {
                if (unsortedArray[i] != unsortedArray.Length-1 && i != 0 && (unsortedArray[i - 1] > unsortedArray[i]))
                {
                    flags++;
                    temp = unsortedArray[i - 1];
                    unsortedArray[i - 1] = unsortedArray[i];
                    unsortedArray[i] = temp;
                }
            }
            if (flags == 0)
            {
                return unsortedArray;
            }
            else { return sortIntArray(unsortedArray); }
        }

        static void Main(string[] args)
        {
            Console.WriteLine("Unsorted array.\r\n");
            for (int i = 0; i < exampleArr.Length; i++) 
            {
                Console.Write(exampleArr[i]);
                if (i != exampleArr.Length-1)
                {
                    Console.Write(", ");
                }
            }
            Console.WriteLine("\r\n\r\n\r\n");
            int[] sortedArr = sortIntArray(exampleArr);
            Console.WriteLine("Sorted array.\r\n");
            for (int i = 0; i < sortedArr.Length; i++)
            {
                Console.Write(sortedArr[i]);
                if (i != sortedArr.Length - 1)
                {
                    Console.Write(", ");
                }
            }
            Console.WriteLine("\r\n");
        }
    }
}

File: IntArray_Sort.php
<?php

$exampleArr = array(
    1, 9, 4, 2, 19, 12
);

function sortIntArray($unsortedArray)
{
    $flags = 0;
    $temp = null;

    for($i = 0; $i < count($unsortedArray); $i++) {
        if($unsortedArray[$i] != count($unsortedArray)-1 && $i != 0 && ($unsortedArray[$i-1] > $unsortedArray[$i])) {

            $flags++;
            $temp = $unsortedArray[$i - 1];
            $unsortedArray[$i - 1] = $unsortedArray[$i];
            $unsortedArray[$i] = $temp;

        }
    }
    if($flags == 0) {
        return $unsortedArray;
    }
    else { return sortIntArray($unsortedArray); }
}

function main($args) {
    echo "Unsorted array.\r\n";
    for($i = 0; $i < count($args); $i++)
    {
        echo $args[$i];
        if($i != count($args) - 1)
        {
            echo ", ";
        }
    }
    echo "\n\n\n";
    $sortedArr = sortIntArray($args);
    echo "Sorted array.\n";
    for($i = 0; $i < count($sortedArr); $i++)
    {
        echo $sortedArr[$i];
        if($i != count($sortedArr) - 1)
        {
            echo ", ";
        }
    }
    echo "\n";
}

main($exampleArr);

?>